src/components/Trade/TradeTabs/Ranges/RangeCardHeader.module.css

```diff
@@ -3,7 +3,7 @@
     /* flex-flow: row nowrap; */
     align-items: center;
     padding: 2px 8px;
-    color: rgba(235, 235, 255, 0.4);
+    color: var(--text2);
```

Comment:
Q: are we still uses these css modules?  Will they get converted to styled components?